src/test/cypress/cypress/integration/FeedTests.spec.js

```diff
@@ -124,4 +124,17 @@ describe('FeedScreen tests', function () {
         cy.loginOpFab('operator1_fr', 'test');
         cy.get('of-light-card').should('have.length', 1);
     });
+
+    it(`Check that a spinner is displayed when the card takes time to load `, function () {
+        cy.sendCard('cypress/cardDetail/cardDetailResponseNotPossible.json');
+        cy.delayRequestResponse('/cards/cards/**')
+        cy.loginOpFab('operator1_fr', 'test');
+
+        // Click on the card
+        cy.get('of-light-card').eq(0).click();
+        
+        cy.checkLoadingSpinnerIsDisplayed();
+        cy.checkLoadingSpinnerIsNotDisplayed();
+
+    });
```

Comment:
I propose to put this test in src/test/cypress/cypress/integration/CardDetail.spec.js instead